domains/tomo.json

```diff
@@ -0,0 +1,11 @@
+{
+  "owner": {
+    "username": "tomocrafter",
+    "email": "tomobotter@yahoo.co.jp",
+    "twitter": "tomocrafter",
+    "discord": "tomocrafter"
+  },
+  "record": {
+    "CNAME": "tomo.pages.dev"
```

Comment:
Seems to just show an example project and not your website, either send a preview of your website or upload your website to Cloudflare pages

Reply:
Hi, Thank you for looking into this pull request. Unfortunately, the web site is in under construction. So I uploaded maintenance page for https://tomo.pages.dev and to prove that cloudflare workers domain is my own.

I hope it verifies. Thank you.